man/systemd.network.xml

```diff
@@ -2435,6 +2424,29 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
         Defaults to <literal>yes</literal>.
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>RelayTarget=</varname></term>
+        <listitem>
+          <para>Takes an IPv4 address, which must be in the format
+          described in
+          <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
+          Turns this DHCP server into a DHCP relay agent. See <ulink url="https://tools.ietf.org/html/rfc1542">RFC 1542</ulink>.
+          The address is the address of DHCP server or another relay agent to forward DHCP messages to and from.</para>
+          Check also BindToInterface= option. Turning it off is required for relaying messages outside.
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>AgentCircuitId=</varname></term>
+        <listitem>
+          <para>Specifies value for Agent Circuit ID sub-option of Relay Agent Information Option. Takes a string.</para>
```

Comment:
Maybe, `RelayAgentCircuitId=` is better?
Also, I think it is better to mention that this will be ignored `RelayTarget=` is not specified, and defaults to unset.